src/styles/main.css

```
@@ -0,0 +1,378 @@
+
+h1, h2, h3, h4, h5, h6 {
+	font-family: 'Raleway', sans-serif;
+	color: var(--primary);
+}
+
+a {
+	color: var(--accent);
+}
+
+code {
+	font-family: Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono,
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: Mono

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)